components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/AsyncLoginsStorage.kt

```diff
@@ -286,6 +287,39 @@ interface AsyncLoginsStorage : AutoCloseable {
      * (IO failure, rust panics, etc).
      */
     fun importLoginsAsync(logins: List<ServerPassword>): Deferred<Long>
+
+    /**
+     * Checks if login already exists and is valid.
+     *
+     * @rejectsWith [InvalidRecordException] On unexpected errors (IO failure, rust panics, etc)
```

Comment:
This comment is incorrect, right? All of `InvalidRecordException` are actually "expected" in case of an invalid login (e.g. they could have been returned, not thrown).

Reply:
Yes, good catch.  Fixing.

Reply:
I coped that line from `mozilla.appservices.logins.LoginsStorage#ensureValid`, we'll need to fix it there as well.